SPECS/zfs.spec

```diff
@@ -575,3 +576,11 @@ systemctl --system daemon-reload >/dev/null || true
 # ignore those files.
 %exclude /usr/share/initramfs-tools
 %endif
+
+%changelog
+* Mon May 06 2024 Yann Dirson <yann.dirson@vates.tech> 2.1.15-2
```

Comment:
Since we are almost 2 months later, I think it would be good to update the date here.

Reply:
done